site/src/constants/mdx-shortcodes.js

```diff
@@ -0,0 +1,6 @@
+import { Link } from 'gatsby';
+import Hero from '../components/hero';
+import SummaryCard from '../components/summary-card';
+import Translations from '../components/translations';
+
+export const SHORTCODES = { Hero, Link, SummaryCard, Translations };
```

Comment:
These are the React components that are available in `.mdx` files.

[Shortcodes](https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#shortcodes)